### PR TITLE
Fix TypeError in parse command

### LINKED ISSUE #5481: scrapy parse -h throws error
### Description
running `scrapy parse -h` from inside of a project throws an error.

### Steps to Reproduce
1.  `scrapy startproject example`
2.  `cd example`
3.  `scrapy parse -h`

**Expected behavior:**  It should show the help message output for the `parse` command
**Actual behavior:**  Throws error and displays traceback information
**Reproduces how often:**  Every time I tried it.

### Versions
v2.6.1

### Additional context

I identified the issue and will submit a PR shortly.

### PR DESCRIPTION
**Pull request by alexpdev**
Fixes #5481 

`scrapy parse -h`   now displays the appropriate help message.

### CODE REVIEW

File: scrapy/commands/parse.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def add_options(self, parser):
         parser.add_argument("--cbkwargs", dest="cbkwargs",
                             help="inject extra callback kwargs into the Request, it must be a valid raw json string")
         parser.add_argument("-d", "--depth", dest="depth", type=int, default=1,
-                            help="maximum depth for parsing requests [default: %default]")
+                            help="maximum depth for parsing requests [default: 1]")
```
Comment (Laerte) [inline scrapy/commands/parse.py:54]:
```suggestion
                            help="maximum depth for parsing requests [default: %(default)s]")
```
This way you don't need to define twice. Syntax is from documentation<sup>[1](https://docs.python.org/3/library/argparse.html)</sup>. Tested locally and it works!

> Replace strings with implicit arguments such as %default or %prog with the standard Python syntax to use dictionaries to format strings, that is, %(default)s and %(prog)s.

**Review by Laerte — CHANGES_REQUESTED** [review]

File: scrapy/commands/parse.py (hunk 1 above)
Comment (alexpdev) [reply scrapy/commands/parse.py]: I fixed thanks for the suggestion.

**Review by wRAR — APPROVED** [review]

**Review by elacuesta — APPROVED** [review]
Looks good, thank you. Two suggestions for the future:
* There's no need to open an issue if you already have a patch in mind, you can submit a PR directly
* Use a descriptive name in the PR. This will (probably) be the message we end up using when merging, and "fixes issue #N" is not good for the history

**Comment by alexpdev** [comment]
> Looks good, thank you. Two suggestions for the future:
> 
> * There's no need to open an issue if you already have a patch in mind, you can submit a PR directly
> * Use a descriptive name in the PR. This will (probably) be the message we end up using when merging, and "fixes issue #N" is not good for the history

Thanks for the advice.

**Review by Laerte — APPROVED** [review]

**Comment by Gallaecio** [comment]
@Laerte Could you rebase this change into the 2.6 branch instead of `master`?

**Comment by Laerte** [comment]
@Gallaecio You want me to cherry-pick the changes from his branch and apply above `2.6` branch? In this case we will need open another PR? (I don't have write access to his branch that's why i thinked on this approach.

**Comment by alexpdev** [comment]
How do i grant write access?

**Comment by Laerte** [comment]
@alexpdev You can follow these steps: https://github.com/scrapy/scrapy/pull/5445#issuecomment-1062664683

(Change the commit id of course)

**Comment by alexpdev** [comment]
> @alexpdev You can follow these steps: [#5445 (comment)](https://github.com/scrapy/scrapy/pull/5445#issuecomment-1062664683)
> 
> (Change the commit id of course)

on the line that starts with `git checkout` I'm getting `fatal 'upstream/2.6' is not a commit and a branch 'master' cannot be created from it`

**Comment by Laerte** [comment]
@alexpdev Just push and change the base here to 2.6.

**Comment by alexpdev** [comment]
Did I do that right?

**Comment by Laerte** [comment]
@alexpdev Yeah, now you just have to change this to 2.6 as well:

Screenshot_20220424-031118-346

**Comment by alexpdev** [comment]
got it

**Comment by Laerte** [comment]
@alexpdev Almost there... As you can see we have commits from `master` in the commits list, you need to drop them and let just yours commits. You can do this:

1.  Rebase from [636127e](https://github.com/alexpdev/scrapy/commit/636127ec1ea2b8949438015c2167ab5d009ff1bf) commit.

```
git rebase -i 636127e^
```

2. This will open a text-editor, you need to **drop** all the commits that are not yours, you can archive this replacing **pick** to **drop**.

Then you need to push again:
```
git push -f
```

**Comment by alexpdev** [comment]
@Laerte Okay, got it... thanks again for all of your help

**Comment by Laerte** [comment]
Nice job! 🚀

**Review by wRAR — APPROVED** [review]

**Pull request merged**